src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -80,6 +80,7 @@ public Elevator() {
     elevNeoMotor1.getPIDController().setP(.144);
     elevNeoMotor1.enableSoftLimit(SoftLimitDirection.kForward, false);
     elevNeoMotor1.enableSoftLimit(SoftLimitDirection.kReverse, false);
+    profiledPid.setTolerance(0.1);
```

Comment:
```suggestion
    profiledPid.setTolerance(Units.inchesToMeters(0.1));
```

Even the tolerance has to be in meters.

We also need to be sure to test to make sure the normal elevator movement doesn't get affected by this tolerance. It shouldn't